lib/restforce/upload_io.rb

```diff
@@ -1,7 +1,5 @@
 # frozen_string_literal: true
 
-require 'faraday/upload_io'
-
 module Restforce
   UploadIO = Faraday::UploadIO
```

Comment:
Does this still work? Is it just the require that is no longer needed?

Reply:
For a better understanding, I am updated the method to the new name of UploadIO
`Faraday::FilePart`

Reply:
I’d be concerned that this might be incompatible with older Faraday versions.

Reply:
Could be in the future. According this faraday PR the method `UploadIO` is deprecated but functional.

I think they will remove that in the future.

https://github.com/lostisland/faraday/pull/1021/files